infra/src/dsl/types/input-types.ts

```diff
@@ -11,6 +11,12 @@ export interface Service {
 export type Hash = { [name: string]: Hash | string | number }
 export type ValueSource = string | ((e: Context) => string)
 export type ValueType = MissingSettingType | ValueSource
+// See https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes for more info
+export type AccessModes =
+  | 'ReadWriteOnce'
+  | 'ReadWriteMany'
+  | 'ReadWriteOncePod'
+  | 'ReadOnlyMany'
```

Comment:
Maybe boil down to what we know will be used right now and in more "dev-terms" (avoid Kubernetes speak)

Reply:
Filter out the modes we will not be using (at least for now)

Reply:
Solved in [602dec1](https://github.com/island-is/island.is/pull/8589/commits/602dec105362ab4194924e03fbccf6a5d5b98589)